PhpcsChanged/CacheManager.php

```diff
@@ -163,6 +168,15 @@ public function addCacheEntry(CacheEntry $entry): void {
 		($this->debug)("Cache add: file '{$entry->path}', type '{$entry->type}', hash '{$entry->hash}', standard '{$entry->phpcsStandard}'");
 	}
 
+	public function getPhpcsStandardCacheKey( string $phpcsStandard, string $warningSeverity, string $errorSeverity ): string {
+		$warningSeverity = '' === $warningSeverity ? '5' : $warningSeverity;
+		$errorSeverity = '' === $errorSeverity ? '5' : $errorSeverity;
+		if ('5' !== $warningSeverity || '5' !==$errorSeverity) {
+			$phpcsStandard .= $warningSeverity . $errorSeverity;
```

Comment:
I wonder if we should add a delimiter between each of these fields, which would make them easier to parse if we ever need to do that, or in case this ever changes to add or remove fields later. Maybe a colon and a letter to label the fields like `VIP:w6:e2`?

Reply:
Yeah, I have been thinking the same. I went with `{standard}:w{w}e{e}` format, since it allow parsing as well, and contains one less colon 😉  Let me know if it works for you.

Reply:
sounds great!